### PR TITLE
change title to Flutter Alterra

### PR DESCRIPTION
**Pull request by kharisazhar**
- Ubah Title Flutter Alterra

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -24,7 +24,7 @@ class MyApp extends StatelessWidget {
         // is not restarted.
         primarySwatch: Colors.blue,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const MyHomePage(title: 'Flutter Alterra'),
```
Comment (achmadfaizalawi) [inline lib/main.dart:27]: Test

File: lib/main.dart [hunk 2]
```diff
@@ -11,7 +11,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
+      title: 'Flutter Demo ',
```
Comment (achmadfaizalawi) [inline lib/main.dart]: Testing

Comment (kharisazhar) [inline lib/main.dart]: Test

**Review by achmadfaizalawi — COMMENTED** [review]
Test

**Comment by kharisazhar** [comment]
test

File: lib/main.dart [hunk 3]
```diff
@@ -24,7 +24,7 @@ class MyApp extends StatelessWidget {
         // is not restarted.
         primarySwatch: Colors.blue,
```
Comment (kharisazhar) [inline lib/main.dart]: ganti warna jadi merah

**Review by kharisazhar — COMMENTED** [review]
please check

File: lib/main.dart (hunk 3 above)
Comment (kharisazhar) [reply lib/main.dart]: gimana jika ganti jadi biru ?

File: lib/main.dart [hunk 4]
```diff
@@ -11,7 +11,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
+      title: 'Flutter Demo ',
       theme: ThemeData(
```
Comment (kharisazhar) [inline lib/main.dart:15]: ganti warna jadi ungu

Comment (kharisazhar) [reply lib/main.dart:15]: file yang mana ?

File: lib/main.dart (hunk 2 above)
Comment (achmadfaizalawi) [inline lib/main.dart]: Ganti flutter jadi demo saja

**Pull request merged**